src/FluentValidation/DefaultValidatorExtensions.cs

```diff
@@ -1085,5 +1085,32 @@ public static class DefaultValidatorExtensions {
 			action(innerValidator.RuleForEach(x => x));
 			return ruleBuilder.SetValidator(innerValidator);
 		}
+
+		/// <summary>
+		/// Defines a enum value validator on the current rule builder that ensures that the specific value is a valid enum value.

```

Comment:
This will need updating to clarify that it's operating against the enum's name rather than the value.